docs/sources/enterprise/usage-insights.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Usage-insights"
+description = "Usage-insights"
+keywords = ["grafana", "usage-insights", "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/"]
+type = "docs"
+[menu.docs]
+name = "Usage-insights"
+parent = "enterprise"
+weight = 700
++++
+
+# Usage Insights
+
+Usage Insights allows you to have a better understanding on how your Grafana instance is used. The collected data are: dashboard views (aggregated and per user), data source errors, and number of data source queries.
+
+> Only available in Grafana Enterprise v7.0+.
+
+## Presence Indicators
+
+Presence indicators are visible to all signed-in users on all dashboards. It shows avatar of users who interacted with the dashboard recently. You can see the user's name by hovering the user's avatar. Users' avatars come from [Gravatar](https://gravatar.com) based on the user's email.
+
+When more users are active on a dashboard than can fit in the presence indicators, click on the `+X` icon that opens the [Dashboard Insights]({{< relref "#dashboard-insights" >}}) drawer and see more details about recent user activity. 
+
+{{< docs-imagebox img="/img/docs/enterprise/presence_indicators.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+You can choose your own definition of "recent" by setting it in the [configuration]({{< relref "../installation/configuration.md">}}) file.
+
+```ini
+[analytics.views]
+# Set age for recent active users
+recent_users_age = 24h
+```
+
+## Dashboard Insights
+
+You can see dashboard usage information by clicking on the `Dashboard insights` button in the top bar.
+
+{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_stats.png" max-width="400px" class="docs-image--no-shadow" >}}{{< docs-imagebox img="/img/docs/enterprise/dashboard_insights_users.png" max-width="400px" class="docs-image--no-shadow" >}}
+
+It shows two kind of information:
+
+- **Stats**: shows the daily query count and the errors count for the last 30 days.
+- **Users & activity**: shows the daily view count for the last 30 days; last activities on the dashboard and recent users (with a limit of 20).
```

Comment:
```suggestion
- **Users & activity:** shows the daily view count for the last 30 days; last activities on the dashboard and recent users (with a limit of 20).
```

Reply:
Is the ampersand actually in the UI? If so, please change it to "and".

Reply:
Yes there is an ampersand in the UI. Is it okay if we leave it that way now and if I change it on the next dashboard insights rework? I added it here: grafana/grafana-enterprise#546

Reply:
Yes, that works